src/sentry/integrations/slack/notify_action.py

```diff
@@ -10,14 +10,19 @@
 
 from .utils import build_attachment
 
+MEMBER_PREFIX = '@'
+CHANNEL_PREFIX = '#'
+strip_channel_chars = ''.join([MEMBER_PREFIX, CHANNEL_PREFIX])
+
 
 class SlackNotifyServiceForm(forms.Form):
     team = forms.ChoiceField(choices=(), widget=forms.Select(
         attrs={'style': 'width:150px'},
     ))
     channel = forms.CharField(widget=forms.TextInput(
-        attrs={'placeholder': 'i.e #critical-errors'},
+        attrs={'placeholder': 'i.e #critical or @evan'},
```

Comment:
Maybe @user would sound better, or a more generic name, is my name generic enough?